python/bro.py

```diff
@@ -2,22 +2,26 @@
 """bro %s -- compression/decompression utility using the Brotli algorithm."""
 
 from __future__ import print_function
-import getopt
+import argparse
 import sys
 import os
 import brotli
 import platform
 
-__usage__ = """\
-Usage: bro [--force] [--decompress] [--input filename] [--output filename]
-    [--mode 'text'|'font'] [--transform]"""
 
-__version__ = '0.1'
+__version__ = '1.0'
 
 
-BROTLI_MODES = {
-    'text': brotli.MODE_TEXT,
-    'font': brotli.MODE_FONT
+# default values of encoder parameters
+DEFAULT_PARAMS = {
+    'mode': brotli.MODE_TEXT,
```

Comment:
I just added a MODE_GENERIC compression mode, which is the default. This avoids the confusion that MODE_TEXT could equally mean 'no idea' and 'text for sure' before. Could you change your argument description and default accordingly?